src/attributes/attr.js

```diff
@@ -3,11 +3,14 @@ define([
 	"../core/access",
 	"./support",
 	"../var/rnotwhite",
+	"./val",
```

Comment:
IMO it's harder to see the logic of the change when it's modifying the previous bad merge.

But however your prefer, just my opinion.